MapboxCoreNavigation/RouteStep.swift

```diff
@@ -1,4 +1,7 @@
 import MapboxDirections
+import Turf
+
+public typealias IntersectionBounds = (entry: Intersection, exit: Intersection)
```

Comment:
I considered it that, however, we have intersection bounds for `motorway`,` toll`, `restricted`, `ferry` or `tunnel`... right? This could apply to these road classes.

Reply:
Also, is it required this typealias be public?

Reply:
nope.